src/engraving/libmscore/engravingitem.h

```diff
@@ -320,6 +320,12 @@ class EngravingItem : public EngravingObject
     virtual Shape shape() const { return Shape(bbox(), this); }
     virtual double baseLine() const { return -height(); }
 
+    virtual mu::RectF clickableBBox() const { return _bbox; }
+    mu::RectF clickablePageBoundingRect() const { return clickableBBox().translated(pagePos()); }
+    Shape clickableShape() const { return Shape(clickableBBox(), this); }
```

Comment:
I think this should be virtual too and return `shape()` instead by default. Otherwise, elements will be clickable in their whole bounding rectangle rather than only in their `shape()`, like currently. One element type where the difference will be very noticeable is multi-measure rests.

Reply:
Good one, thanks!